examples/address_verification.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'smile-identity-core'
+# See https://docs.usesmileid.com/products/for-individuals-kyc/address-verification for
+# more information on address verification
+
+# Initialize
+partner_id = '' # login to the Smile Identity portal to view your partner id
+api_key = '' # copy your API key from the Smile Identity portal
+sid_server = '0' # Use 0 for the sandbox server, use 1 for production server
+
+connection = SmileIdentityCore::AddressVerification.new(partner_id, api_key, sid_server)
+
+request_params = {
+  country: 'ZA', # The country [NG, ZA] where the address is located
+  address: 'Cape town', # The address to be verified
+  # utility_number: "12345678911", # The utility number to be verified specifically for NG
```

Comment:
I think you could improve this example. I have no idea looking at this which fields are required and which ones are optional. Which fields are mutually exclusive? Do you utility numbers and id numbers fit into a specific format? What is included in the response?

Reply:
This has been addressed